pkg/client/unversioned/request.go

```diff
@@ -440,15 +442,15 @@ func (r *Request) VersionedParams(obj runtime.Object, convertor runtime.ObjectCo
 		for _, value := range v {
 			// TODO: Move it to setParam method, once we get rid of
 			// FieldSelectorParam & LabelSelectorParam methods.
-			if k == unversioned.LabelSelectorQueryParam(r.groupVersion.String()) && value == "" {
+			if k == unversioned.LabelSelectorQueryParam(r.content.GroupVersion.String()) && value == "" {
```

Comment:
It was just the convention we agreed on.  The two are equivalent and
we were using one fairly consistently.